test/generator/template_values_test.rb

```diff
@@ -22,13 +22,13 @@ def test_version
 
     def test_exercise_name
       expected_exercise_name = 'alpha_beta'
-      subject = TemplateValues.new(@arguments.merge(exercise_name: 'alpha-beta'))
+      subject = TemplateValues.new(@arguments.merge(exercise_name: expected_exercise_name))
       assert_equal expected_exercise_name, subject.exercise_name
     end
```

Comment:
I agree. There are three tests of this type in this test suite. I would not normally include such a test, but as two already existed, I converted rather than deleted...